react-todo-app/src/components/TodoContainer.js

```diff
@@ -1,12 +1,107 @@
-import React from "react"
-class TodoContainer extends React.Component {
-  render() {
-    return (
-      <div>
-        <h1>Hello from Create React App</h1>
-        <p>I am in a React Component!</p>
-      </div>
+import React, { useState, useEffect } from "react"
+import { Route, Switch } from "react-router-dom"
+import Header from "./Header"
+import InputTodo from "./InputTodo"
+import TodosList from "./TodosList";
+import { v4 as uuidv4 } from "uuid";
+
+import About from "../pages/About"
+import NotMatch from "../pages/NotMatch"
+import Navbar from "./Navbar";
+
+const TodoContainer = () => {
+
+  const [todos, setTodos] = useState(getInitialTodos())
+
+  // useEffect(() => {
+  //   // getting stored items
+  //   const temp = localStorage.getItem("todos")
+  //   const loadedTodos = JSON.parse(temp)
+
+  //   if (loadedTodos) {
+  //     setTodos(loadedTodos)
+  //   }
+  // }, [])
```

Comment:
It would be better if you do not commit old pieces of code as comments.
if you need to review a previous version of your code, you can always use `git history`.